057-AzureRouteServer/Coach/README.md

```diff
@@ -0,0 +1,72 @@
+# What The Hack - Azure Route Server - Coach Guide
+
+## Introduction
+
+Welcome to the coach's guide for the Azure Route Server What The Hack. Here you will find links to specific guidance for coaches for each of the challenges.
+
+This hack includes an optional [lecture presentation](Lectures.pptx) that features short presentations to introduce key topics associated with each challenge. It is recommended that the host present each short presentation before attendees kick off that challenge.
+
+**NOTE:** If you are a Hackathon participant, this is the answer guide. Don't cheat yourself by looking at these during the hack! Go learn something. :)
+
+## Coach's Guides
+
+- Challenge 00: **[Prerequisites - Ready, Set, GO!](./Solution-00.md)**
+	 - Prepare your workstation to work with Azure.
+- Challenge 01: **[Building a Basic Hub and Spoke Topology utilizing a central Network Virtual Appliance](./Solution-01.md)**
+	 - Warming up
+- Challenge 02: **[Introduce Azure Route Server and peer with a Network Virtual appliance](./Solution-02.md)**
+	 - Azure Route Server basic concepts, interaction with NVAs
+- Challenge 03: **[Connect Network Virtual Appliance to an SDWAN environment](./Solution-03.md)**
+	 - Azure Route Server interaction with Virtual Network Gateways (VPN or ExpressRoute)
+- Challenge 04: **[Introduce a High Availability with Central Network Virtual Appliances](./Solution-04.md)**
+	 - High Availability concepts
+
+## Coach Prerequisites
+
+This hack has pre-reqs that a coach is responsible for understanding and/or setting up BEFORE hosting an event. Please review the [What The Hack Hosting Guide](https://aka.ms/wthhost) for information on how to host a hack event.
+
+The guide covers the common preparation steps a coach needs to do before any What The Hack event, including how to properly configure Microsoft Teams.
+
+### Student Resources
+
+Before the hack, it is the Coach's responsibility to download and package up the contents of the `/Student/Resources` folder of this hack into a "Resources.zip" file. The coach should then provide a copy of the `Resources.zip` file to all students at the start of the hack. 
+
+In Challenge 00, the students will run a script from this Resources.zip file that deploys a baseline Hub & Spoke network topology into their Azure subscriptions with Linux VMs.  
```

Comment:
I re-worded this section to refer to the coach needing to provide username & password to the students.  However, I think this should be removed from this section.  This belongs in the coach guide for Challenge 00, not here on the main page.

Also, I see that you have hard coded this username/password combo into the "HubAndSpoke.azcli" script file, and then instruct the coach to share with the student.  This officially commits this combo into the WTH repo, which will likely be flagged by GitHub's security bot.  It's a better practice to make the password be parameterized and then change the instructions in C0 for the student to supply a password of their choice when they run the deployment script.

Reply:
I'm not clear on this item. What part is better to be moved to C0? Also, I like the idea of being specific about student supplying their own username and password

Reply:
Also, the reason I put stuff on the main page is because I was using the below as an example, it looks quite similar. Please let me know what I need to change :) https://github.com/maurojasmsft/WhatTheHack/tree/xxx-AzureRouteServer/001-IntroToKubernetes/Coach

Reply:
Edited as per discussion with @maurojasmsft